src/Evolution/Systems/Cce/Components/WorldtubeBoundary.hpp

```diff
@@ -91,7 +87,7 @@ struct H5WorldtubeBoundary
   using base_type::execute_next_phase;
   using base_type::initialize;
   using typename base_type::chare_type;
-  using typename base_type::const_global_cache_tag_list;
+  using const_global_cache_tags = tmpl::list<Tags::InitializeJ>;
```

Comment:
These aliases should probably `tmpl::push_back` to the `base_type::const_global_cache_tag_list`

Reply:
the base currently does not have a `const_global_cache_tags` -- the one it had was incorrectly called `const_global_cache_tag_list` and didn't have anything the actions didn't have anyway, so isn't needed.

Reply:
probably better to delete it when you get the chance

Reply:
yep, the commit in question already does that. (lines 42-45)